osf/management/commands/dark_ia_item.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+import internetarchive
+
+
+class Command(BaseCommand):
+    help = '"darks" an Internet Archive item, making it invisible to all users, use this to aid in ' \
+           'withdraw requests or remove spam'
+
+    def add_arguments(self, parser):
+        parser.add_argument('guids', type=str, nargs='+', help='The GUIDs of the items to darken.')
+        parser.add_argument('comment', type=str, help='The comment for explaining the darkening action.')
+        parser.add_argument('version', type=str, help='Version of the item. Default is "v1".')
+        parser.add_argument('access', type=str, help='AWS access key for our collection')
+        parser.add_argument('secret', type=str, help='AWS secret key for our collection')
+
+    def handle(self, *args, **kwargs):
+        guids = kwargs['guids']
+        comment = kwargs['comment']
+        access_key = kwargs['access']
+        secret_key = kwargs['secret']
```

Comment:
If the goal is to be able to run this automatically on spam confirmation, or manually via admin, these would need to be available via settings, rather than input. 

In order to avoid this and the additional requirement, it might be better to implement as a pigeon API endpoint, as OSF can already talk to Pigeon, and Pigeon can already talk to IA.

Reply:
(Likewise, there should also be a reversing call to `.undark` available, if that route is taken)